models/points_test.go

```diff
@@ -163,41 +163,22 @@ func TestParsePointWhitespaceValue(t *testing.T) {
 	}
 }
 
-func TestParsePointSingleEquals(t *testing.T) {
-	pts, err := models.ParsePointsString("=")
-	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. expected error`, "=")
-	}
-
-	if exp := 0; len(pts) != exp {
-		t.Errorf(`ParsePoints("%s") len mismatch. got %v, exp %v`, "", len(pts), exp)
-	}
-}
-
 func TestParsePointNoFields(t *testing.T) {
-	_, err := models.ParsePointsString("cpu_load_short,host=server01,region=us-west")
-	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu_load_short,host=server01,region=us-west")
-	}
-
-	_, err = models.ParsePointsString("cpu")
-	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu")
-	}
-
-	_, err = models.ParsePointsString("cpu,")
-	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu,")
-	}
-
-	_, err = models.ParsePointsString("cpu, value=1")
-	if err == nil {
-		t.Errorf(`ParsePoints("%s") mismatch. got nil, exp error`, "cpu, value=1")
-	}
-
-	_, err = models.ParsePointsString("cpu,,, value=1")
```

Comment:
Yes it is covered by the `cpu,` case, but I didn't intend to leave it out. I've added it back (you can never have enough test cases :laughing:).

I gave them all the once over and added a couple more.